src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -0,0 +1,256 @@
+import React, {useMemo, useCallback, useRef, useState} from 'react';
+import {ActivityIndicator, View} from 'react-native';
+import Button from '@components/Button';
+import ConfirmModal from '@components/ConfirmModal';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import * as Expensicons from '@components/Icon/Expensicons';
+import * as Illustrations from '@components/Icon/Illustrations';
+import MenuItem from '@components/MenuItem';
+import MenuItemList from '@components/MenuItemList';
+import ScreenWrapper from '@components/ScreenWrapper';
+import ScrollView from '@components/ScrollView';
+import Section from '@components/Section';
+import ThreeDotsMenu from '@components/ThreeDotsMenu';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+// import useWaitForNavigation from '@hooks/useWaitForNavigation';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import Navigation from '@libs/Navigation/Navigation';
+import type {AnchorPosition} from '@styles/index';
+import variables from '@styles/variables';
+import CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
+import type IconAsset from '@src/types/utils/IconAsset';
+
+type WorkspaceMenuItem = {
+    translationKey?: TranslationPaths;
+    descriptionTranslationKey?: TranslationPaths;
+    icon?: IconAsset;
+    iconRight?: IconAsset;
+    iconHeight?: number;
+    iconWidth?: number;
+    interactive?: boolean;
+    disabled?: boolean;
+    onButtonPress?: () => void;
+};
+
+function PolicyAccountingPage() {
+    const theme = useTheme();
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    // const waitForNavigate = useWaitForNavigation();
+    const {isSmallScreenWidth, windowWidth} = useWindowDimensions();
+
+    const [threeDotsMenuPosition, setThreeDotsMenuPosition] = useState<AnchorPosition>({horizontal: 0, vertical: 0});
+    const [policyIsConnectedToAccountingSystem, setPolicyIsConnectedToAccountingSystem] = useState(false);
+    const [isSyncInProgress, setIsSyncInProgress] = useState(false);
+    const [isDisconnectModalOpen, setIsDisconnectModalOpen] = useState(false);
+    const threeDotsMenuContainerRef = useRef<View>(null);
+
+    // fake a QBO connection sync
+    const openQBOsync = useCallback(
+        () => {
+        setIsSyncInProgress(true);
+        setTimeout(() => setIsSyncInProgress(false), 5000);
+        setPolicyIsConnectedToAccountingSystem(true);
+    }, []);
+
+    const connectionIconSize = useMemo(() => ({iconHeight: variables.avatarSizeNormal, iconWidth: variables.avatarSizeNormal}), []);
```

Comment:
No need for this object, just define the sizes inline. And use `variables.iconSizeExtraLarge` instead since those are not avatars